syncto/tests/test_functional.py

```diff
@@ -0,0 +1,81 @@
+import mock
+from uuid import uuid4
+
+from cliquet.errors import ERRORS
+from cliquet.tests.support import FormattedErrorMixin
+
+from syncto import AUTHORIZATION_HEADER, CLIENT_STATE_HEADER
+
+from .support import BaseWebTest, unittest
+
+
+COLLECTION_URL = "/buckets/syncto/collections/{collection_id}/records"
```

Comment:
We can, the name here doesn't really matter. After discussion with Tarek, I state that they are two cases:

- When an app handle user personal data, it should use ``default``
- When an app handle shared data, it is a good idea to use the app name as the bucket name.